layer.md

```diff
@@ -23,7 +23,7 @@ Removals are represented using "[whiteout](#whiteouts)" file entries (See [Repre
 
 ### File Types
 
-Throughout this document section, the use of word "files" includes:
+Throughout this document section, the use of word "files" or "entries" includes:
```

Comment:
“of word” is awkward.  Maybe:

> Throughout this document section, the words “files” and “entries” include:

I'm also not sure about “document section”.  I expect we want something scoped to the whole layer.md, not just the “File Types” h3.  How about:

> Throughout the [layer documentation](#image-layer-filesystem-changeset), the words “files” and “entries” include: